### ISSUE TITLE
Question about imbedding images

### ISSUE THREAD

**Issue by Yucheng98** [cfss-fa20/hw01#19]
On the guide, they inserted the picture using a URL, yet I did not succeed using that code. I then uploaded a picture to my repository, and directly typed the picture's name as the code. Now on my side it is showing the picture when I preview my code. But since the picture is in my repository, and while TA and professor are viewing my document, will the code be able to retrieve the picture from my repository? Could you please let me know if it is the correct way to imbed a picture? 
Thank you so much!

**Comment by djanvier** [comment]
Hi there! Yes, we should be able to view the image. You can also check by pushing your most recent changes to GitHub and viewing the README.md from your repository. Whatever is in your repository is what we will be viewing/grading. I can also check for you if you push your changes. Hope that helps!

**Comment by Yucheng98** [comment]
Thank you for the reply! However I think something is still wrong. When I preview my code locally, the picture was shown. Now I have pushed it into my repository, the picture becomes the code I typed. Could you please help me check? Thank you!

**Comment by djanvier** [comment]
Hi! Yes, it looks like the code is not rendering correctly. Would you be able to either screenshot or copy and paste the code you used so I can take a look?

**Comment by Yucheng98** [comment]
<img width="466" alt="d55590082412e981171b5f4b9e95ed2" src="https://user-images.githubusercontent.com/72301511/95006837-22b42300-063b-11eb-882a-16f9cca70560.png">

**Comment by Yucheng98** [comment]
Above is the code I wrote, and when I preview it, the picture showed
<img width="494" alt="b88317e262b47c95e14ab5b3269abe8" src="https://user-images.githubusercontent.com/72301511/95006848-3cee0100-063b-11eb-88d2-05a4991bca5a.png">

**Comment by djanvier** [comment]
Hmm, that's weird. First, I would make sure that you end the line before the image (ending in "carved") with two spaces. That way R Markdown knows to start a new paragraph. I say this because there is no new paragraph in the rendered version on GitHub, but your code suggests that there should be. If that doesn't work, just let me know and we can go from there.

**Comment by Yucheng98** [comment]
I started the code for the image in a new line, but it still is not working. 
<img width="361" alt="2c568b5a0a3d297b1230f9656f65c96" src="https://user-images.githubusercontent.com/72301511/95007069-d9b19e00-063d-11eb-8e91-efdb702ef2a6.png">

**Comment by djanvier** [comment]
Yeah, I just saw that :(. My guess is that is has something to do with the way GitHub renders the Markdown syntax. It might have something to do with the **image name** since there's a space. You might want to rename the image with either an underscore or no space, and then try again.

**Comment by Yucheng98** [comment]
Now a new problem arises.... After I changed the picture name, I failed to push.

To https://github.com/Yucheng98/hw01.git
 ! [rejected]        HEAD -> master (fetch first)
error: failed to push some refs to 'https://github.com/Yucheng98/hw01.git'
hint: Updates were rejected because the remote contains work that you do
hint: not have locally. This is usually caused by another repository pushing
hint: to the same ref. You may want to first integrate the remote changes
hint: (e.g., 'git pull ...') before pushing again.
hint: See the 'Note about fast-forwards' in 'git push --help' for details.

I was not sure what means by integrate remote changes. How do I synchronize the remote and the local?

**Comment by djanvier** [comment]
For that error, I would reference issue #18

**Comment by djanvier** [comment]
TLDR: any changes made to your GitHub repo need to be **pulled** to your local repo so that everything is the same. Otherwise, it will generate an error.

**Comment by Yucheng98** [comment]
Thank you so much for your help! I changed the name of the picture by deleting the space, and I think it is showing on github as well. Could you please help me check again? Thank you!!

**Comment by bensoltoff** [comment]
It works now..and yes I think GitHub does not like files with spaces in their names. We'll talk more about this in a few weeks when we cover code style.

**Comment by Yucheng98** [comment]
Thank you Dominique and Professor Soltoff!
